### PR TITLE
fix: Convert Linux help to markdown

### LINKED ISSUE #286: chore: Update Linux help for 14.0
This is a general list of help documentation to update for 14.0 (`staging` branch). We'll have a separate list for each product

- [x] Try to reorganize to match Windows help hierarchy
- [x] If editing files, convert to Markdown
- [x] Continue Markdown migration and address TODOs (in the keymanapp repo)

### PR DESCRIPTION
**Pull request by darcywong00**
Fixes #286 converting the Linux help to markdown and reorganizing the pages.

reference/ pages left intact in php (tied to man page content)

### CODE REVIEW

**Comment by darcywong00** [comment]
actually, I'll add some more pages for welcome and what's new

File: products/linux/14.0/common/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Q. How do I install Keyman for Linux?
+**A.** In Ubuntu and Wasta-Linux, Keyman for Linux can be installed via launchpad:
+```bash
+sudo add-apt-repository ppa:keymanapp/keyman
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get install keyman onboard-keyman
+```
+
+Keyman for Linux packages are also available in any current Debian at:
```
Comment (ermshiperete) [inline products/linux/14.0/common/index.md:14]:
```suggestion
An older version of Keyman for Linux packages is also available in any current Debian at: 
```

File: products/linux/14.0/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* [What's New](about/whatsnew)
+* [Version History](about/history)
+
+## [Getting Started](start/)
+* [Installing a keyboard](start/installing-keyboard)
+
+## [Common Questions](common/)
+
+## Ask for Help
+[Ask other users in the SIL Keyman Keyman Community](https://community.software.sil.org/c/keyman)
```
Comment (ermshiperete) [inline products/linux/14.0/index.md:18]:
```suggestion
[Ask other users in the SIL Keyman Community](https://community.software.sil.org/c/keyman)
```

File: products/linux/14.0/start/installing-keyboard.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+title: Installing Keyboards with Keyman for Linux
+---
+
+Keyboards are installed with Keyman Configuration. From the launcher, enter `Keyman Configuration`.
```
Comment (ermshiperete) [inline products/linux/14.0/start/installing-keyboard.md:5]:
```suggestion
Keyboards are installed with Keyman Configuration. From the launcher, enter `Keyman Configuration`.
Alternatively from the command line run `km-config`.
```

File: products/linux/14.0/start/installing-keyboard.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+![](../linux_images/package-khmer-angkor-welcome.png "Package khmer_angkor")
+
+The keyboard should appear in Keyman Configuration.
+
+![](../linux_images/km-config-khmer-angkor.png "Keyman Configuration khmer_angkor")
+
+### Install the Input Source in Region & Language Settings
+This section is only relevant for Ubuntu. At this point the keyboard should be ready to use on Wasta.
+
+**Note:** Keyman Configuration only installs the keyboard into IBus. In order to use the keyboard in Ubuntu,
```
Comment (ermshiperete) [inline products/linux/14.0/start/installing-keyboard.md:31]: Is this still true? I think it should get added automatically Ubuntu 18.04+.

**Review by ermshiperete — APPROVED** [review]

File: products/linux/14.0/start/installing-keyboard.md (hunk 4 above)
Comment (darcywong00) [reply products/linux/14.0/start/installing-keyboard.md:32]: oh yeah - I can remove this section then.
You've made Keyman for Linux a lot more user-friendly than when this was written 😄

**Pull request merged**